.github/workflows/ci.yaml

```diff
@@ -78,12 +78,7 @@ jobs:
     - name: Run tests
       run: |
         sed -i 's/config.eager_load = .*/config.eager_load = true/' config/environments/test.rb
-        bundle exec rails db:schema:load --trace
-        mv db/schema.rb db/schema.rb.original
-        bundle exec rails db:rollback STEP=4
-        bundle exec rails db:migrate
-        sh -c "[ '$DB' != 'sqlite' ] || diff db/schema.rb db/schema.rb.original "
-        bundle exec rails db:test:prepare
+        bundle exec rails db:setup
```

Comment:
@manno   

Comment regarding https://github.com/frab/frab/pull/870/commits/7fc92db9857a1aea36bac8872e8b376950060cbd


I think we should just remove the `diff` lines. The rollback+migrate test can stay. They are useful for making sure the migrations actually work.

Reply:
it seems to be fixed now. thanks